FirebaseAppDistribution/Sources/Public/FIRAppDistribution.h

```diff
@@ -13,24 +13,9 @@
 // limitations under the License.
 
 @class FIRAppDistributionRelease;
-#import <Foundation/Foundation.h>
```

Comment:
Was this delete intended?  It may still be needed for NSError's declaration.

Reply:
I couldn't tell initially why we were importing Foundation because it builds either way, but good point on NSError. I'll add it back.